.gitignore

```diff
@@ -12,3 +12,9 @@ dist/
 htmlcov/
 .coverage
 test-results.xml
+
+.DS_Store
+
+.idea/
```

Comment:
Please put system and editor specific ignores in your user's ignore file. E.g.
```
# ~/.gitconfig
[core]
  excludesfile = ~/.gitignore
```
and then 
```
# ~/.gitignore
.DS_Store
.idea
.vscode
```

Reply:
updated it